### PR TITLE
Change the interface to use DegreesOfFreedom and adjust the C# accordingly

### PR DESCRIPTION
**Pull request by pleroy**

### CODE REVIEW

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 1]
```diff
@@ -475,13 +466,13 @@ public partial class PluginAdapter : UnityEngine.MonoBehaviour {
     Log.Info("active vessel orbit.vel : " + active_vessel.orbit.vel);
     Log.Info("active vessel GetVel : " + active_vessel.orbit.GetVel());
     Log.Info("Principia orbit.pos : " + 
-             (Vector3d)VesselDisplacementFromParent(
+             (Vector3d)VesselFromParent(
                  plugin_,
-                 active_vessel.id.ToString()));
+                 active_vessel.id.ToString()).q);
```
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: fold?

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 2]
```diff
@@ -475,13 +466,13 @@ public partial class PluginAdapter : UnityEngine.MonoBehaviour {
...
     Log.Info("Principia orbit.vel : " + 
-             (Vector3d)VesselParentRelativeVelocity(
+             (Vector3d)VesselFromParent(
                  plugin_,
-                 active_vessel.id.ToString()));
+                 active_vessel.id.ToString()).p);
```
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: same?

Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 1 above)
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

**Pull request merged**